### ISSUE TITLE
createdAt / modifiedAt incorrectly reported as invalid

### ISSUE THREAD

**Issue by jmcanterafonseca** [FIWARE/context.Orion-LD#43]
Using

https://raw.githubusercontent.com/FIWARE/dataModels/master/specs/AgriFood/AgriCrop/example-normalized-ld.jsonld 

an error is raised

{"type":"http://uri.etsi.org/ngsi-ld/errors/BadRequestData","title":"Attribute must be a JSON object","details":"createdAt"}

createdAt / modifiedAt are not reified properties, thus they are directly the date as string in ISO8601 format

**Comment by kzangeli** [comment]
True, those two "builtins" aren't getting any special treatment by orionld as of today.
Not sure what the broker should do if encountered with them ...
There are really only three options:
1. Return Error
2. Ignore them (as the broker sets these values itself)
3. Use them and let them override the builtin values.

From this issue I assume we will go with option 3.
Do we really want that?
I mean, the timestamps will be incorrect ...

**Comment by kzangeli** [comment]
PR https://github.com/FIWARE/context.Orion-LD/pull/48 should fix this problem (option 3: builtin values are overridden)

**Comment by jmcanterafonseca** [comment]
From the specification:

**5.2.2 Common members**
The JSON-LD representation of NGSI-LD Entity, Property, Relationship, Context Source Registration and Subscription
can include the common members described by table 5.2.2-1.
Those members are read-only, and shall be automatically generated by NGSI-LD implementations. They shall not be provided by Context Producers. In the event that they are provided (in update or create operations) NGSI-LD implementations **shall ignore them.**

**Comment by kzangeli** [comment]
ok, I'll change what I implemented to IGNORE instead of override.
Makes sense, no probs

**Comment by kzangeli** [comment]
github closed the issue for me ... reopening

**Comment by kzangeli** [comment]
The issue should be fixed by PR #64

**Comment by jmcanterafonseca** [comment]
still crashing

```
{
    "branch": "(HEAD",
    "kbase version": "0.2",
    "kalloc version": "0.2",
    "kjson version": "0.2",
    "boost version": "1_62",
    "microhttpd version": "0.9.48-0",
    "openssl version": "OpenSSL 1.1.0j  20 Nov 2018",
    "mongo version": "1.1.3",
    "rapidjson version": "1.0.2",
    "libcurl version": "UNKNOWN",
    "libuuid version": "UNKNOWN"
}
```
